src/sql/migrations.rs

```diff
@@ -690,6 +690,19 @@ CREATE INDEX smtp_messageid ON imap(rfc724_mid);
         }
         sql.set_db_version(98).await?;
     }
+    if dbversion < 99 {
+        sql.execute_migration(
+            "DROP INDEX msgs_index5; -- Drop index on `starred` table
+             ALTER TABLE msgs DROP COLUMN server_folder;
+             ALTER TABLE msgs DROP COLUMN server_uid;
+             ALTER TABLE msgs DROP COLUMN move_state;
+             ALTER TABLE msgs DROP COLUMN starred;
```

Comment:
i'd leave the column starred, there are ideas about adding that. if it then turns out, the column is not needed, we can drop it then.

Reply:
i am not thinking much about IMAP wrt starring, maybe it makes sense, maybe not. i would just not decide that now and leave the column until we have implemented starring.